calculator/models.py

```diff
@@ -18,6 +18,17 @@ class Client(models.Model):
     phone = models.BigIntegerField("Phone")
     cpf = models.BigIntegerField("CPF", unique=True)
 
+    @property
+    def is_indebted(self):
```

Comment:
The business rule is: If a client contracted a loan in the past and paid all but missed more than 3 monthly payments or didn’t pay all the loan, you need to deny the new one.

I think it is missing to check if any loan was not paid out completely. For example, the loan has 4 installments, was created 4 months ago and therefore should be paid, if not, the customer is in debit.